cirq-core/cirq/protocols/circuit_diagram_info_protocol.py

```diff
@@ -325,10 +325,15 @@ def _op_info_with_fallback(
     op: 'cirq.Operation', args: 'cirq.CircuitDiagramInfoArgs'
 ) -> 'cirq.CircuitDiagramInfo':
     info = protocols.circuit_diagram_info(op, args, None)
+    rows = (
+        tuple(op.qubits)
+        + tuple(protocols.measurement_key_objs(op))
```

Comment:
This should only happen for the cbits we're actually drawing. That needs to go into the diagram info somewhere.